ddl/index.go

```diff
@@ -654,7 +654,9 @@ func (w *worker) doBackfillIndexTask(t table.Table, colMap map[int64]*types.Fiel
 
 	startTime := time.Now()
 	var ret *taskResult
-	err := kv.RunInNewTxn(w.ctx.GetStore(), true, func(txn kv.Transaction) error {
+	store := w.ctx.GetStore()
+	store.SetPriority(kv.PriorityLow)
```

Comment:
Will this affect the global priority of the store instance.

Reply:
Like https://github.com/pingcap/tidb/blob/master/meta/meta.go#L89

Reply:
set txn.GetSnapshot() priority is ok.